docs/source/quick_start.rst

```diff
@@ -34,11 +34,25 @@ the *keyring*. Here's an example of a decrypted keyring file ::
 
     {
       "lochness": {
+        "REDCAP": {
+          "example": {
+            "redcap.example": [
+              "example"
+            ]
+          }
+        },
         "SECRETS": {
           "StudyA": "quick brown fox jumped over lazy dog"
         }
       },
 
+      "redcap.StudyA": {
```

Comment:
Again, for all other modules, we have `xnat.example` like. Why do we have `redcap.StudyA` here?

Reply:
my bad - let me fix it

Reply:
https://github.com/PREDICT-DPACC/lochness/pull/27/commits/aa5f99b091aae5ca3fa09ef6e92d1bb3c528f30e

fixed here

Reply:
https://github.com/PREDICT-DPACC/lochness/pull/27/commits/a5355163dcf0078133b1b895c3b79f15f35e411c

and here too